<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewcoursesofastudent</class>
 <widget class="QDialog" name="viewcoursesofastudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="students_cmb">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a Student</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QListWidget" name="inProgress_lv"/>
   <widget class="QListWidget" name="finished_lv"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Finished Courses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewCourses_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>View Courses</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>In Progress Courses</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
